### ISSUE TITLE
Allow resend verification email without time limit until 3rd try

### ISSUE THREAD

**Issue by lyqht** [GeeksHacking/flagit-docs#37]
User gets this error immediately if they try to re-login without verifying.

![CleanShot 2024-07-19 at 22 32 23@2x](https://github.com/user-attachments/assets/ca034ba0-aa3a-4efa-8bdb-f0089a06600a)

**Comment by joeltio** [comment]
I don't think this is something we can change as Firebase is the one who sets the rate-limits. Closing as not feasible, but feel free to open if there's a way around it
